### PR TITLE
Minor visual tweaks

### PR DESCRIPTION
**Pull request by sircharlo**

### CODE REVIEW

**Review by sircharlo — COMMENTED** [review]
- Remove bg color for fields in settings (they look disabled otherwise)
- Center and pad meeting date in media list header
- Vertically center action buttions in media list
- Change color of Test App button
- Made settings panels accordion instead of popout

File: src/renderer/components/media/MediaControls.vue [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
         </v-btn>
       </v-col>
       <v-col class="text-center d-flex justify-center">
-        <v-btn color="secondary" :disabled="mediaActive" @click="clearDate()">
+        <v-btn class="px-3" color="secondary" :disabled="mediaActive" @click="clearDate()">
           {{ date }}
         </v-btn>
       </v-col>
-      <v-col class="text-right" cols="auto">
+      <v-col class="text-right" cols="4">
```
Comment (mtdvlpr) [inline src/renderer/components/media/MediaControls.vue:16]: Although I agree that centered looks better, it doesn't work when you enable PowerPoint shortcuts and are on the lowest width.

**Review by mtdvlpr — CHANGES_REQUESTED** [review]
Nice tweaks! One small thing that needs to be different.

File: src/renderer/components/media/MediaControls.vue (hunk 1 above)
Comment (mtdvlpr) [reply src/renderer/components/media/MediaControls.vue:16]: Either we have to move some of those control buttons, increase the min width or do some tweaks to above values.

Comment (mtdvlpr) [reply src/renderer/components/media/MediaControls.vue:16]: Screenshot from 2022-09-27 18-23-49

Comment (sircharlo) [reply src/renderer/components/media/MediaControls.vue:16]: What then if we moved the folder name to the left, and have the other icons on the right?

File: src/renderer/pages/index.vue [hunk 2]
```diff
@@ -462,6 +462,10 @@ export default Vue.extend({
         5: 'accent',
         6: 'accent',
       }
+      this.dayColors[this.$getPrefs('meeting.mwDay')]  =
+          'secondary'
+      this.dayColors[this.$getPrefs('meeting.weDay')]  =
+          'secondary'
```
Comment (mtdvlpr) [inline src/renderer/pages/index.vue:468]: Doesn't work for the first week, when days of the week are missing. This should fix it:

```suggestion
      if (this.currentWeek === this.$dayjs().isoWeek()) {
        this.setDayColor(this.$getPrefs('meeting.mwDay'), 'secondary')
        this.setDayColor(this.$getPrefs('meeting.weDay'), 'secondary')
      } else {
        this.dayColors[this.$getPrefs('meeting.mwDay')] = 'secondary'
        this.dayColors[this.$getPrefs('meeting.weDay')] = 'secondary'
      }
```

File: src/renderer/assets/scss/main.scss [hunk 3]
```diff
@@ -3,6 +3,15 @@ html {
...
 }
 
+.page-enter-active,
+.page-leave-active {
+  transition: opacity 0.1s;
+}
+.page-enter,
+.page-leave-to {
+  opacity: 0;
+}
```
Comment (mtdvlpr) [inline src/renderer/assets/scss/main.scss:13]: You added the classes, but I don't think you're using them anywhere. Am I wrong?

File: src/renderer/components/media/MediaControls.vue (hunk 1 above)
Comment (mtdvlpr) [reply src/renderer/components/media/MediaControls.vue:16]: How do you feel about this?
Screenshot from 2022-09-27 18-49-38

Comment (sircharlo) [reply src/renderer/components/media/MediaControls.vue:16]: Nice. I like it

Comment (mtdvlpr) [reply src/renderer/components/media/MediaControls.vue]: I pushed it to your branch, so you'll need to do a `git pull`

File: src/renderer/assets/scss/main.scss (hunk 3 above)
Comment (sircharlo) [reply src/renderer/assets/scss/main.scss:13]: They're used when pages are changed. For example, when going from home to settings

Comment (mtdvlpr) [reply src/renderer/assets/scss/main.scss:13]: Ah, it's a nuxt feature. Nice!

**Review by mtdvlpr — APPROVED** [review]

**Review by mtdvlpr — APPROVED** [review]

**Review by mtdvlpr — APPROVED** [review]

**Pull request merged**